docs/_content/ontwikkelaars/algemeen/zgw-api-landschap-grenzen.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "ZGW API-landschap grenzen"
+date: '07-06-2019'
+weight: 50
+---
+
+In het Common Ground API-landschap worden straks alle bronnen netjes ontsloten 
+middels een API. Er is een API die Kadastergegevens ontsluit, een API voor 
+Zaken, Documenten, Personen, Medewerkers, etc.
+
+Op dit moment wordt er hard gewerkt aan een set API's die bij elkaar de
+kern vormen voor Zaakgericht werken, de ZGW API's. Deze bestaan op het moment
```

Comment:
Consequent spreken van "API's voor Zaakgericht werken"

Reply:
In plaats van ZGW API's of?